README.md

```diff
@@ -9,35 +9,44 @@ This should support using categories with *arr apps to isolate downloads as long
 ## How to use the script
 
 1. Run these commands in a folder qBittorrent have access
-```
+```shell
 git clone https://github.com/gravelfreeman/qbit-hardlinker
 ```
-```
+
+```shell
 chmod +x qbit-hardlinker.sh
 ```
 
 2. Edit the destination path, and base_path for your hardlinked files, base path is likely the same as your "default save path"
 
-```
+```shell
 nano qbit-hardlinker.sh
 ```
 
-3. Modify the variables on line 7 & 12  and save ``CTRL+X``
+3. Modify the variables on line 7 & 12  and save `CTRL+X`
 
 > Caution! Destination folder must be on the same dataset and mount path.
 
 4. qBittorent > Tools > Options > Downloads Tab and scroll down to *Run external program* section and check the box *Run external program on torrent finished*
 
-5. Add the script path, ``%R``  in the text field to match the example below
+5. Add the script path, `%F` in the text field to match the example below
+
+```
+/configs/qbit-hardlinker.sh "%F"
+```
+
+If all your torrents always have folders you may use `%R` instead of `%F`.
+
+From the settings page:
 
-````
-/configs/qbit-hardlinker.sh "%R"
-````
+> %F: Content path (same as root path for multi-file torrent)
+>
```

Comment:
For some reason if I didn't have this it would collapse the block quote to a single line
(╯°□°）╯︵ ┻━┻